src/app/contextBroker/contextBroker.cpp

```diff
@@ -1472,6 +1489,17 @@ int main(int argC, char* argV[])
   paParse(paArgs, argC, (char**) argV, 1, false);
   lmTimeFormat(0, (char*) "%Y-%m-%dT%H:%M:%S");
 
+  //
+  // FIXME P8: for release 0.24, -reqMutexPolicy != "all" ("all" is default) cannot
```

Comment:
I think that we have an issue number for this. Maybe it could be references here. However, I understand that that issue is the next thing after merging this PR, so probably it doesn't worth the effort...